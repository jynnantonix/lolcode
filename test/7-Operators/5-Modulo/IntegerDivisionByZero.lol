This test makes sure that integer modulo by 0 raises an error.

HAI
	MOD OF 1 AN 0
KTHXBAI
