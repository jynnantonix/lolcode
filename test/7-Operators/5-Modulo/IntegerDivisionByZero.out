Unexpected error: integer division or modulo by zero
